templates/common/_base/units/ovs-vswitchd.service.yaml

```diff
@@ -4,3 +4,6 @@ dropins:
     contents: |
       [Service]
       Restart=always
+      ExecStartPre=-/bin/sh -c '/usr/bin/chown -R :$${OVS_USER_ID##*:} /var/lib/openvswitch'
```

Comment:
Are you sure about the leading `:` to only change the group?
There's quite a lot of syntatic sorcery going on here.
OK `$$` is to quote it from systemd's expansion, so the shell sees it.
The `##*:` is...I don't quite follow that, what is it doing?

Reply:
I leveraged this sorcery from an existing ovs-vswitchd service file on a deployed cluster. I am assuming it works :).